jac-uk/admin
# #1596 Eligibility report pagination

**Author checklist**


---
## What's included?
Add pagination items to the bottom of the table. See [JAC Kit #57 Added pagination to bottom of table](https://github.com/jac-uk/jac-kit/pull/57). 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Preview URL: https://jac-admin-develop--pr1605-feature-1596-eligibi-y6h4b37j.web.app

- Make sure you use `@jac-uk/jac-kit` version `0.1.0`
- Open an exercise with eligibility issues in it
- Check pagination functionality 

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Demo:

![jac-admin-eligibility-issues-pagination](https://user-images.githubusercontent.com/79906532/161984669-65988e05-5cc1-455f-8cab-b0abb87bad71.gif)

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
